src/Network/Network/PrivateLinkService/PrivateEndpointConnection/SetAzurePrivateEndpointConnection.cs

```diff
@@ -0,0 +1,71 @@
+﻿using AutoMapper;
+using Microsoft.Azure.Commands.Network.Models;
+using Microsoft.Azure.Commands.ResourceManager.Common.Tags;
+using Microsoft.Azure.Management.Network;
+using System;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using MNM = Microsoft.Azure.Management.Network.Models;
+using Microsoft.WindowsAzure.Commands.Common.CustomAttributes;
+namespace Microsoft.Azure.Commands.Network
+{
+    [Cmdlet(VerbsCommon.Set, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "PrivateEndpointConnection"), OutputType(typeof(PSPrivateLinkService))]
+    public class SetAzurePrivateEndpointConnection : PrivateLinkServiceBaseCmdlet
+    {
+        [Alias("ResourceName")]
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The resource name.")]
+        [ValidateNotNullOrEmpty]
+        public string Name { get; set; }
+
+        [Parameter(
+           Mandatory = true,
+           ValueFromPipelineByPropertyName = true,
+           HelpMessage = "The private link service name.")]
+        [ValidateNotNullOrEmpty]
+        public string ServiceName { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The resource group name.")]
+        [ResourceGroupCompleter]
+        [ValidateNotNullOrEmpty]
+        public virtual string ResourceGroupName { get; set; }
+
+        [Parameter(
+           Mandatory = true,
+           ValueFromPipelineByPropertyName = true,
+           HelpMessage = "Approved or rejected the resource.")]
+        [PSArgumentCompleter("Approved","Rejected")]
```

Comment:
From API specs description it also has "Removed". Should this value also be mapped there?